main.py

```
@@ -28,6 +40,61 @@
+    try:
+        import urllib.request
+        import json
+        api_url = os.environ.get("TRAFFIC_API_URL", "http://127.0.0.1:8000/api/traffic-snapshots/")
+        payload = {k: v for k, v in row.items() if k != "timestamp"}
+        payload["timestamp"] = row["timestamp"]
+        req = urllib.request.Request(
+            api_url,
+            data=json.dumps(payload).encode("utf-8"),
+            headers={"Content-Type": "application/json"},
+            method="POST",
+        )
```

Ruff (0.14.14)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)